README.md

```diff
@@ -333,7 +333,12 @@ Also with *yaml* support using [optional features][]:
 
     pip install confight[yaml]
 
+Similarly, for *[hcl][]* support:
+
+    pip install confight[pyhcl]
```

Comment:
I guess this should be (as defined [here](https://github.com/Avature/confight/pull/17/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R29))

```suggestion
    pip install confight[hcl]
```

Reply:
Thanks, fixed!